ivi-sdk-java/src/main/java/games/mythical/ivi/sdk/client/IVIPaymentClient.java

```diff
@@ -100,18 +100,12 @@ public List<IVIPaymentMethod> getPaymentMethods(
         String token,
         PaymentProviderId paymentProviderId
     ) throws IVIException {
-        // only supported for Cybersource for now
-        if(paymentProviderId != PaymentProviderId.CYBERSOURCE) {
-            return Collections.emptyList();
-        }
-
```

Comment:
So. I could just approve this.
But.. They might be used to getting 200s back with empty lists.
It would be safer to do the weird thing and add GR4VY to the list of supported payment providers.